src/main.rs

```diff
@@ -1,35 +1,22 @@
-extern crate image;
-extern crate dssim;
-extern crate imgref;
-extern crate lodepng;
-use std::env;
-use std::path::{Path};
-use image::{open, ImageResult, DynamicImage};
-use imgref::{ImgVec};
-use dssim::*;
+extern crate optimizer;
```

Comment:
I would move main inside a bin folder and have lib directly in the source folder, so we can do
```rs
extern crate optimal_image
```
and keep the lib name consistent with the crate

what you think?